src/utils/cql/evaluateCql.js

```diff
@@ -0,0 +1,65 @@
+const { Client } = require('cql-translation-service-client');
+const { Library, Executor, Repository } = require('cql-execution');
+const { PatientSource } = require('cql-exec-fhir');
+const atob = require('atob');
+const axios = require('axios');
+const { ValueSetLoader } = require('./ValueSetLoader');
+
+// TODO: Add url to config
+const client = new Client(
+  'http://moonshot-dev.mitre.org:8080/cql/translator?annotations=true&result-types=true'
+);
+
+async function evaluateCQL(resources, expression, library, patientId) {
+  const elm = await fetchELM(library);
+  const valueSetLoader = new ValueSetLoader(library);
+  const valueSetMap = await valueSetLoader.seedValueSets();
+  const result = executeElm(resources, elm, valueSetMap);
+  return result.patientResults[patientId][expression] === 1;
```

Comment:
This should be `=== true` since we expect the expressions to be boolean-valued. 
```suggestion
  return result.patientResults[patientId][expression] === true;
```